mpitests/test_apply.py

```diff
@@ -34,9 +34,12 @@
 )
 
 
-def create_random_init_state(numWires, R_DTYPE, seed_value=48):
+def create_random_init_state(numWires, C_DTYPE, seed_value=48):
```

Comment:
UPPER or lower? Why do we have this inconsistency in code base? Is that the same for other devices :grimacing:

Reply:
Yes, you are right. I will move everything to lower case to keep consistency with the rest of the devices.